models/remote_test_data.go

```diff
@@ -0,0 +1,48 @@
+package models
```

Comment:
Placed it inside 
`src/github.com/almighty/almighty-core/test/remote_test_data.go`